### DISCUSSION TITLE
[Bug] Weird display issue with `Code` button

### DISCUSSION THREAD

**Discussion by Deadlyelder** [community/community#9666] [Other Feature Feedback, Questions, & Ideas]
Saw it happening multiple times today, the code button on the repo (used to get information about cloning or downloading the zip) tends to lose the arrow that would other exist.

I was only able to replicate this when turning off the color blind feature. Works fine on color blind theme, while for normal view  occurs rather randomly (and only with certain repos). Furthermore, the overlay popup of the button messes up the text.

Attached is a screenshoot.

![Screen Shot 2022-01-05 at 4 28 00 PM](https://user-images.githubusercontent.com/6140893/148243470-968c500d-c950-480e-bdd4-e4f619da2b72.png)
![Screen Shot 2022-01-05 at 4 30 50 PM](https://user-images.githubusercontent.com/6140893/148243948-f0f9c083-df10-42d5-8f88-3689492dd316.png)

**Comment by MathiasReker** [comment]
Hello. You can report the bug here: https://support.github.com/contact/bug-report
